src/main/java/com/unzer/payment/CustomerCompanyData.java

```diff
@@ -20,35 +20,122 @@
  * #L%
  */
 
+import com.google.gson.annotations.SerializedName;
+import com.unzer.payment.communication.json.JSonCompanyInfo;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class CustomerCompanyData {
-	public enum RegistrationType {REGISTERED, NOT_REGISTERED}
-
-	private RegistrationType registrationType;
-	private String commercialRegisterNumber;
-	private CommercialSector commercialSector;
-	
-	public RegistrationType getRegistrationType() {
-		return registrationType;
-	}
-
-	public void setRegistrationType(RegistrationType registrationType) {
-		this.registrationType = registrationType;
-	}
-
-	public String getCommercialRegisterNumber() {
-		return commercialRegisterNumber;
-	}
-
-	public void setCommercialRegisterNumber(String commercialRegisterNumber) {
-		this.commercialRegisterNumber = commercialRegisterNumber;
-	}
-
-	public CommercialSector getCommercialSector() {
-		return commercialSector;
-	}
-
-	public void setCommercialSector(CommercialSector commercialSector) {
-		this.commercialSector = commercialSector;
-	}
+    public CompanyType getCompanyType() {
+        return companyType;
+    }
+
+    public void setCompanyType(CompanyType companyType) {
+        this.companyType = companyType;
+    }
+
+    public Owner getOwner() {
+        return owner;
+    }
+
+    public void setOwner(Owner owner) {
+        this.owner = owner;
+    }
+
+    public enum RegistrationType {REGISTERED, NOT_REGISTERED}
+
+    private RegistrationType registrationType;
+    private String commercialRegisterNumber;
+    private CommercialSector commercialSector;
+    private CompanyType companyType;
+    private Owner owner;
```

Comment:
- It looks like the "function" property was missing here before, maybe it would make sence to add it or create a ticket for later.
- I would place all properties on top of the class before any functions.